Bottom: 6bbc2f526b97dfafc16e7ec34dc8e4e1ce587f40
Top:    232a5020b2d128a616b93f4b14bc5a527a330b7c
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-18 16:51:58 -0700

Refresh of rs-opts

---

diff --git a/include/rdma/rsocket.h b/include/rdma/rsocket.h
index 5310d33..87ee943 100644
--- a/include/rdma/rsocket.h
+++ b/include/rdma/rsocket.h
@@ -72,6 +72,8 @@ int rselect(int nfds, fd_set *readfds, fd_set *writefds,
 int rgetpeername(int socket, struct sockaddr *addr, socklen_t *addrlen);
 int rgetsockname(int socket, struct sockaddr *addr, socklen_t *addrlen);
 
+#define SOL_RDMA 0x10000
+
 int rsetsockopt(int socket, int level, int optname,
 		const void *optval, socklen_t optlen);
 int rgetsockopt(int socket, int level, int optname,
diff --git a/src/rsocket.c b/src/rsocket.c
index de0ec3e..95957d7 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1661,6 +1661,8 @@ int rsetsockopt(int socket, int level, int optname,
 			break;
 		}
 		break;
+	case SOL_RDMA:
+		break;
 	default:
 		break;
 	}
@@ -1726,6 +1728,9 @@ int rgetsockopt(int socket, int level, int optname,
 			break;
 		}
 		break;
+	case SOL_RDMA:
+		ret = ENOTSUP;
+		break;
 	default:
 		ret = ENOTSUP;
 		break;
